packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -177,10 +177,7 @@ class _CodeViewState extends State<CodeView>
     // middle third of the screen.
     if (parsedScript.lineCount * CodeView.rowHeight > extent) {
       final lineIndex = location.line - 1;
-      final scrollPosition = lineIndex * CodeView.rowHeight -
-          (widget.controller.shouldCenterScrollLocation
```

Comment:
why are we no longer centering the scroll location?

Reply:
Oops, I meant to always center the location, not top align. Fixed.